maintainers/maintainer-list.nix

```diff
@@ -6178,6 +6178,18 @@
     githubId = 138074;
     name = "Pedro Pombeiro";
   };
+  poscat = {
```

Comment:
This should be in a seperate commit. Speaking of commits, the others should be squashed